server/controllers/admin/families/getCampaigns.ts

```diff
@@ -0,0 +1,24 @@
+import { NextFunction, Response, Request } from 'express';
+import { Campaign, Family } from '../../../database/models';
+import { CustomError, paramsSchema } from '../../../utils';
+
+const getCampagins = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { id } = await paramsSchema.validateAsync(req.params);
+    const response = await Campaign.findAll({
+      attributes: ['title'],
+      include: { model: Family, attributes: [], where: { id } },
+    });
+    if (!response.length) {
+      throw new CustomError("Family doesn't exist", 400);
```

Comment:
If there is no family then nothing wrong with the request, there is just no data